Cargo.toml

```diff
@@ -23,22 +23,25 @@ repository = "https://github.com/contentauth/c2patool"
 
 [dependencies]
 anyhow = "1.0"
-c2pa = { version = "0.32.0", features = [
+# TODO: temp use ok-nick/roi branch since our fixtures are signed using features from 2.1 spec
```

Comment:
what do you mean here? The SDK doesn't have support for the 2.1 spec yet, so what features are you talking about?

Reply:
Some of the assets are signed with the region of interest assertion using an "identified" field which was introduced in the 2.1 spec. I noticed this as well when implementing the assertion for c2pa-rs. We may want to resign our fixtures without that field.